lms/djangoapps/grades/subsection_grade_factory.py

```diff
@@ -107,8 +107,9 @@ def update(self, subsection, only_if_higher=None, score_deleted=False, force_upd
             self._update_saved_subsection_grade(subsection.location, grade_model)
 
             if settings.FEATURES.get('ENABLE_COURSE_ASSESSMENT_GRADE_CHANGE_SIGNAL'):
-                COURSE_ASSESSMENT_GRADE_CHANGED.send_robust(
```

Comment:
after reading docs for these, not clear to me why we choose send vs send_robust here.. is there an exception handling behavior you are after?

Reply:
I was more wanting to reflect the other signals we have to the integrated channels. Happy to keep it `send_robust` but we aren't catching any exceptions in the receiver so what do you think?

Reply:
yeah if you want consistency with other signalling used it's best to keep consistent, seems like exception handling is different between the two, robust may work unless you are afraid of losing any uncaught exceptions